Relativity.SearchTermsReport.SDK.md

```diff
@@ -4,10 +4,27 @@
 
 This package contains interfaces for the public APIs of the Relativity SearchTermsReport application.
 
-## v13.6.14
+## v13.7.31
 
 ### Release Notes
-* Added CancelJob, IndexContainsLongTextField, and GetRetry endpoints

```

Comment:
Is there a reason why this was removed?

Reply:
There is no GetRetry endpoint.  Unsure where that came from.  Also we never published 13.6.14 to NuGet so i'm just trying to get this somewhat current and will publish 13.6.10 and 13.7.31 once this gets approved.